README.md

```diff
@@ -28,15 +28,16 @@ New features are regularly announced in [changelogs](https://intellij-rust.githu
 The plugin is compatible with all IntelliJ-based IDEs starting from the version 2022.2, with the following differences in the sets of the available features:
 
 
-|                        | Open-source and Educational IDEs<sup>*</sup> | [CLion] (commercial) | [IntelliJ IDEA] Ultimate, [PyCharm] Professional, [GoLand] (commercial) | [WebStorm], [PhpStorm], other commercial IDEs |
-|------------------------|---|---|---|---|
-| Language support       | + | + | + | + |
-| Cargo support          | + | + | + | + |
-| Code coverage          | + | + | + | + |
-| Debugger               | - | + | +** | - |
-| Profiler               | - | + | - | - |
-| Valgrind Memcheck      | - | + | - | - |
-| [Detecting duplicates] | - | + | + | + |
+|                        | Open-source and Educational IDEs<sup>*</sup> | [CLion] (commercial) | [IntelliJ IDEA] Ultimate, [GoLand] (commercial) | [PyCharm] Professional | [WebStorm], [PhpStorm], other commercial IDEs |
+|------------------------|----------------------------------------------|---|---|------------------------|---|
+| Language support       | +                                            | + | + | + | + |
+| Cargo support          | +                                            | + | + | + | + |
+| Code coverage          | +                                            | + | + | + | + |
+| Debugger               | -                                            | + | +** | +** | - |
+| Profiler               | -                                            | + | - | - | - |
+| Run targets            | -                                            | + | + | - | - |
+| Valgrind Memcheck      | -                                            | + | - | - | - |
+| [Detecting duplicates] | -                                            | + | + | + | + |
```

Comment:
Let's do one of the following things:
- use `|---|` as width of each column
- use variable width depending on a column header text. `Reformat table` quick fix may help here

Otherwise, it's looks confusing

Reply:
Committed the changes. Thanks for noting!